### ISSUE TITLE
Possible misinterpretation of note in §11.3.1.2 Inline Regions.

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/ttml2#592]
The note:

> A content element can only be associated with a single region. Consequently, if a content element specifies a region attribute, then any explicit inline region specification is ignored.

is incorrect - we have established through discussion of other issues that some content elements are associated with multiple regions. I think this is about explicit association by specifying a `region` attribute. Suggest rewording to:

_A content element can only **specify** a single region. Consequently, if a content element specifies a region attribute, then any explicit inline region specification is ignored._

**Comment by palemieux** [comment]
The PR was closed before the 14 day period was over. I do not have a concern with the contents of the note. It should however not be a note since it sets a normative precedence order.

**Comment by skynavga** [comment]
This issue and PR was marked editorial. I have just reviewed the details of the change (again) and it is indeed editorial. Therefore, we have followed the rules.

**Comment by skynavga** [comment]
And, no, it does not set a precedence order, normative or not.

**Comment by skynavga** [comment]
Just a reminder, an introduction, change, or deletion of a note is, by definition, editorial, since all notes are non-normative.

**Comment by palemieux** [comment]
Where is the normative prose that specifies that _explicit inline region specification is ignored_, if not in this note?

**Comment by skynavga** [comment]
I can keep closing this all night. Please file another issue if you wish.

**Comment by nigelmegitt** [comment]
> Where is the normative prose that specifies that explicit inline region specification is ignored, if not in this note?

@palemieux I'm content that this is normatively stated, since the first step in [process inline regions] is:

> 1. if the [attributes] information item property of B contains a region attribute, then exit this procedure

So I agree this is editorial.
